uwsgi.ini

```diff
@@ -1,12 +1,17 @@
 [uwsgi]
+; Set up uwsgi HTTP router on the specified address with HTTP
 http = :5000
+http-keepalive=1
+http-timeout=90
```

Comment:
Why 90? The GCP load balancer timeout is 30 seconds, so this should only be a bit longer than that (something like 35 seconds).